--- plotters-0.3.5/Cargo.toml	1970-01-01T00:00:01+00:00
+++ plotters-0.3.5/Cargo.toml	2026-01-29T18:30:48.558216+00:00
@@ -137,25 +137,6 @@
     "pathfinder_geometry",
 ]
 
-[target."cfg(all(target_arch = \"wasm32\", not(target_os = \"wasi\")))".dependencies.wasm-bindgen]
-version = "0.2.62"
-
-[target."cfg(all(target_arch = \"wasm32\", not(target_os = \"wasi\")))".dependencies.web-sys]
-version = "0.3.51"
-features = [
-    "Document",
-    "DomRect",
-    "Element",
-    "HtmlElement",
-    "Node",
-    "Window",
-    "HtmlCanvasElement",
-    "CanvasRenderingContext2d",
-]
-
-[target."cfg(all(target_arch = \"wasm32\", not(target_os = \"wasi\")))".dev-dependencies.wasm-bindgen-test]
-version = "0.3.24"
-
 [target."cfg(not(all(target_arch = \"wasm32\", not(target_os = \"wasi\"))))".dependencies.ab_glyph]
 version = "0.2.12"
 optional = true
@@ -198,3 +179,4 @@
 
 [target."cfg(not(target_arch = \"wasm32\"))".dev-dependencies.rand_xorshift]
 version = "0.3.0"
+
