--- perfetto-recorder-0.3.2/Cargo.toml	2006-07-24T01:21:28+00:00
+++ perfetto-recorder-0.3.2/Cargo.toml	2026-03-18T10:39:59.539811+00:00
@@ -65,13 +65,7 @@
 version = "0.30.1"
 features = ["process"]
 
-[target.'cfg(target_os = "macos")'.dependencies.libc]
-version = "0.2.176"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61.2"
-features = ["Win32_System_Threading"]
-
 [profile.opt-debug]
 debug = 2
 inherits = "release"
+
